<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pie</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>720</y>
      <width>1020</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playToggleButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cutBeginButton">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>720</y>
      <width>25</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>[</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cutEndButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>720</y>
      <width>25</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousFrameButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>720</y>
      <width>25</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextFrameButton">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>720</y>
      <width>25</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>|&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
